resources/client/cl_bank.ts

```diff
@@ -35,7 +36,7 @@ onNet(BankEvents.TRANSACTION_ALERT, (result: IAlertProps) => {
   SendNuiMessage(
     JSON.stringify({
       app: 'BANK',
-      method: 'setAlert',
+      method: PhoneEvents.SET_ALERT,
```

Comment:
Dont use a global one if we are not changing the UI too, because we will be handling the same alert once per app repeated! Use scoped for now